modules/internal/LocaleModelHelpSetup.chpl

```diff
@@ -233,6 +233,12 @@ module LocaleModelHelpSetup {
   proc helpSetupLocaleGPU(dst: borrowed LocaleModel, out local_name:string,
       numSublocales: int, type GPULocale){
 
+    // Cylic distribution uses this variable to determine how many
+    // data-parallel tasks to have per locale. If this gets set to 0 then we
+    // end up not processing things in the first locale.
+    extern proc chpl_task_getMaxPar(): uint(32);
+    dst.maxTaskPar = chpl_task_getMaxPar();
```

Comment:
I am surprised that lack of this only breaks Cyclic.

Reply:
It probably does break other distributions. I only tested cyclic so that's the one I referred to. I can update the comment to just say "Cyclic (and likely other distributions) use this variable ..."